lib/utils/eol-conversion.ts

```diff
@@ -14,11 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import mmmagic = require('mmmagic');
```

Comment:
I've moved these imports inside the relevant functions, mainly so that the `mmmagic` module doesn't even need to be loaded if `process.platform !== 'win32'`.

Reply:
@pdcastro Interesting, I guess these distributions have no `file` command.  Thanks for the quick fix!  I suppose we could also consider testing for `file` on non-win OSs, in case there is any value in having the functionality on linux (e.g. if you've copied windows format files for some reason).

Reply:
Just leaving note here that user had problem on Fedora 31 and Debian 9.11